speechbrain/decoders/transducer.py

```diff
@@ -64,7 +64,7 @@ class TransducerBeamSearcher(torch.nn.Module):
     ...     blank_id=0
     ... )
     >>> dec = sb.nnet.RNN.GRU(
-    ...     hidden_size=5, input_shape=(1, 40, 34), bidirectional=True
+    ...     hidden_size=10, input_shape=(1, 40, 34), bidirectional=False
```

Comment:
Why this change ?

Reply:
The predictor is autoregressive, can't use a bidirectional net. (It's just a docstring but it's misleading)